pkgs/top-level/all-packages.nix

```
@@ -15547,7 +15547,7 @@ with pkgs;
   lld_15 = llvmPackages_15.lld;
   lld_16 = llvmPackages_16.lld;
 
-  lldb = llvmPackages_latest.lldb;
+  lldb = lldb_14;
```

`lldb` was [an outlier](https://github.com/NixOS/nixpkgs/pull/116736) in that unlike the other tool aliases (`clang`, `lld`, etc) it pointed to `llvmPackages_latest` instead of `llvmPackages`.

I believe the motivation (#99365) was to have a newer `lldb` version by default; if we're planning to bump `llvmPackages` more aggressively I think we can safely set this to:

```suggestion
  lldb = llvmPackages.lldb;
```
Though this would mean that for now `lldb` in nixpkgs will go backwards from v14 to v11/v12 (depending on platform); I'm not sure if that's acceptable.